src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -29,11 +29,12 @@
     <!-- The default level of a check is "error". -->
     <!-- See http://checkstyle.sourceforge.net/config.html#Severity -->
     <property name="severity" value="error" />
-    <property name="ignoreLines" value="2"/>
     <property name="fileExtensions" value="java"/>
     <property name="header"
         value="${checkstyle.headerCheck.content}"
-        default="/**\n * Copyright (c) 2010-2017 by the respective copyright holders.\n *\n * All rights reserved. This program and the accompanying materials\n * are made available under the terms of the Eclipse Public License v1.0\n * which accompanies this distribution, and is available at\n * http://www.eclipse.org/legal/epl-v10.html\n */"/>  </module>
+        default="^/\*\*$\n^ \* Copyright \(c\) \d\d\d\d-\d\d\d\d by the respective copyright holders\.$\n^ \*$\n^ \* All rights reserved\. This program and the accompanying materials$\n^ \* are made available under the terms of the Eclipse Public License v1\.0$\n^ \* which accompanies this distribution, and is available at$\n^ \* http://www.eclipse.org/legal/epl\-v10\.html$"/>
```

Comment:
The only drawback is that it still does not allow us to validate years, which is a common failure, its was better than it was.